canonical-web-and-design/maas-ui
# Upload scripts

## Done
Allow uploading of testing and commissioning scripts.

## QA
1. Visit either testing or commissioning scripts.
2. Click "Upload script"
3. Ensure, Cancel takes you back to the appropriate script list view.
4. Click "Upload script" again.
5. Drag and drop a valid script (text file < 2MB)
6. Click upload
7. You should be redirected to the scripts list view, see your script in the list, and see a success message. Ensure the contents of the script are displaying when clicking the script name.
8. Click "Upload script" again.
9. Try out various cases for invalid files (> 2MB, wrong mimetype)
10. Try dragging dropping more than one file etc
You should see appropriate errors explaining whats supported.



## Also Fixes
https://github.com/canonical-web-and-design/maas-ui/issues/184
